packages/pigeon/lib/generator_tools.dart

```diff
@@ -432,3 +432,39 @@ Iterable<EnumeratedClass> getCodecClasses(Api api, Root root) sync* {
 /// Returns true if the [TypeDeclaration] represents an enum.
 bool isEnum(Root root, TypeDeclaration type) =>
     root.enums.map((Enum e) => e.name).contains(type.baseName);
+
+/// Formats documentation comments and adds them to current Indent.
```

Comment:
I think it's worth elaborating here about `open` and `close`, providing the examples of what they would be for `///` and `/** */`-style comments. Also, consider calling them `openCommentToken` and `closeCommentToken` so they are more self-documenting.

Reply:
I think you'll also need a `continuationToken` (or similar name), since the standard way of writing the `/** */` multi-line comment is with a `*` on each new line (see https://google.github.io/styleguide/javaguide.html#s7.1-javadoc-formatting for an example).